packages/web-components/fast-ssr/README.md

```diff
@@ -190,6 +190,34 @@ ElementRenderer.disable("my-element");
 templateRenderer.render(html`<my-element></my-element>`);
 ```
 
+### Hydration
+#### `defer-hydration` Attribute
+The `defer-hydration` attribute is an attribute added to each custom-element rendered by the `ElementRenderer`. The presence of the attribute indicates to client-side code that the element should not hydrate it's view. When the attribute is removed, the element is free to hydrate itself.
+
+This attribute is added automatically during custom element rendering to all FAST custom elements. If your app does not require orchestrating element hydration, emission of this attribute can be prevented by configuring the renderer:
+```ts
+const { templateRenderer } = fastSSR({deferHydration: false});
+```
+
+> For more information on this community-protocol, see https://github.com/webcomponents-cg/community-protocols/pull/15
+#### Configuring FAST-Element
+`@microsoft/fast-element` must be configured to respect the `defer-hydration` attribute. To do this, simply import the install code into the client-side application before defining the custom elements:
+```ts
+import "@microsoft/fast-element-hydration";
```

Comment:
Should this match an export path?

Reply:
yes, good catch. Got ahead of myself there